src/main/java/jag/UiCLI.java

```diff
@@ -0,0 +1,337 @@
+package jag;
+
+import java.util.Scanner;
+
+/**
+ * Represents an instance of User Interface interactions
+ * Purpose is to read messages / commands, respond appropriately,
+ * and return values to be used for other Command instances e.g. "Event"
+ */
+public class UiCLI extends Ui {
```

Comment:
Since abbrieviations and acronyms should not be uppercase when used as part of a name, perhaps this could be renamed to something like UiCli?